source/Octopus.Tentacle/Scripts/ScriptWorkspace.cs

```diff
@@ -30,17 +30,16 @@ public class ScriptWorkspace : IScriptWorkspace
             fileSystem.EnsureDiskHasEnoughFreeSpace(workingDirectory);
         }
 
-        protected virtual string BootstrapScriptName => "Bootstrap.ps1";
+        const string BootstrapScriptFileName = "Bootstrap.ps1";
+        protected virtual string BootstrapScriptName => BootstrapScriptFileName;
 
         const string LogFileName = "Output.log";
-        public string LogFilePath => GetLogFilePath(WorkingDirectory);
+        public string LogFilePath => Path.Combine(WorkingDirectory, LogFileName);
```

Comment:
Why did this change?

Reply:
The static `GetLogFilePath` wasn't needed outside of this class, so I removed the static method and changed this to use it's logic.